Dockerfile.cockpit

```diff
@@ -0,0 +1,9 @@
+FROM fedora:latest
```

Comment:
Use `FROM welder/fedora:latest` this should already be downloaded as an intermediate layer of the e2e image. This way you don't have to pull the latest container of the upstream Fedora image.

Reply:
The cockpit image built from welder/fedora:latest does not work in my local env and Travis. All of cases are failed. In my local env, the welder-web is not able to be loaded in iframe. Travis result: https://travis-ci.org/henrywang/welder-web/jobs/270048364
Since the web-cockpit image is built from cache almost all the time, IMO, we just keep fedora:latest in this PR and add a Trello card to investigate the root cause.

Reply:
ok, leave it as-is for now and create a new card to tackle this later